documentation/eclipse.asciidoc

```diff
@@ -22,6 +22,7 @@ You may also supply additional arguments as `devon eclipse «args»`. These are
 |`ws-re[verse]`           |reverse merge changes from workspace into settings
 |`ws-reverse-add`         |reverse merge adding new properties
 |`create-script`          |create launch script for this IDE, your current workspace and your OS
+|`import «type» [«path»]` |import Maven/Plain projects from the current directory or a given path
```

Comment:
Does this optional `path` need to be an absolute path name or can it be relative? If it must be absolute, this must be mentioned here.

I have the impression, that it could be relative, hence even a "." might work. But I neither have checked it yet myself. Nor do I get an answer from the [documentation](https://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.oomph.util.doc%2Fjavadoc%2Forg%2Feclipse%2Foomph%2Fresources%2FSourceLocator.html&anchor=getRootFolder()), I have found so far.